Sources/ArcGISToolkit/Components/BasemapGallery/BasemapGallery.swift

```diff
@@ -25,11 +25,11 @@ public struct BasemapGallery: View {
         /// The `BasemapGallery` will display as a grid when there is an appropriate
         /// width available for the gallery to do so. Otherwise, the gallery will display as a list.
         /// Defaults to `125` when displayed as a list, `300` when displayed as a grid.
-        case automatic(listWidth: CGFloat = 125, gridWidth: CGFloat = 300)
+        case automatic
         /// The `BasemapGallery` will display as a grid. Defaults to `300`.
-        case grid(width: CGFloat = 300)
+        case grid
         /// The `BasemapGallery` will display as a list. Defaults to `125`.
-        case list(width: CGFloat = 125)
+        case list
     }
```

Comment:
The associated values are no longer needed because we can just use the width from the geometry reader now.